docs/installation/env_requirement.rst

```diff
@@ -86,8 +86,7 @@ For python dependencies,
 
 .. note::
 
-    - For Android build, `ANDROID_NDK_HOME` must be confifigured by using ``export ANDROID_NDK_HOME=/path/to/ndk``
+    - For Android build, `ANDROID_NDK_HOME` must be confifigured by using ``export ANDROID_NDK_HOME=/path/to/ndk``.
```

Comment:
Thanks! BTW, maybe you can help fixing `confifigured` to `configured`.

Reply:
Yes, okay.

Reply:
LGTM, okay to **squash** and merge.